### PR TITLE
Update jsonld js and add safe mode support.

### PR DESCRIPTION
**Pull request by davidlehn**
- Updates jsonld.js to 8.0.0.
- Add safe mode option UI.
- Show error details if available.  Hopefully this doesn't have degenerate cases involving huge errors or recursive serialization.  A built project would use other libraries to avoid that but this is difficult for now.
- Disables signature tabs since old jsonld-signatures no longer works with newer jsonld.js.  Bundled code and default suites are not distributed.  Need to move this to a build process if that functionality is to be kept.

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
Thanks!

We should try to get the signature stuff working again with ed25519-signature-2020 as quickly as we can.

**Comment by gkellogg** [comment]
It would be useful to have a reference and note to motivate the purpose of “safe mode”.

**Comment by davidlehn** [comment]
> It would be useful to have a reference and note to motivate the purpose of “safe mode”.

Yes, this is happening out of the intended order due to limited time.  This was one of the initial [RFCs](https://github.com/json-ld/json-ld.org/issues/778) I was going to write up.  I imagine there will be feedback and this first implementation will be updated as needed.

**Comment by davidlehn** [comment]
> We should try to get the signature stuff working again with ed25519-signature-2020 as quickly as we can.

Arguably, the signature code should be on some other playground.  It's not strictly a json-ld thing.  In any case, the whole playground should be rewritten with modern tools at some point, and that's not a "quick" project.  And deployment would no longer be simple static files.  The only "quick" solution I've considered is to just abuse npm and npm cdns with a site specific playground build packages for the whole thing or just parts like jsigs support.

**Comment by gkellogg** [comment]
> > We should try to get the signature stuff working again with ed25519-signature-2020 as quickly as we can.
> 
> Arguably, the signature code should be on some other playground.

I agree, I think the JSON-LD playground should remain focused on JSON-LD. Things that make use of the technology, such as signatures, probably should go somewhere else. If they can be hosted off of GitHub pages directly, that might make it easier to spin-up a playground in other repos.

> It's not strictly a json-ld thing. In any case, the whole playground should be rewritten with modern tools at some point, and that's not a "quick" project. And deployment would no longer be simple static files. ...

The YAML-LD group has been asking for a YAML-LD playground; a complicating factor is the limitations of the current UI (specifying YAML-LD vs JSON-LD inout or output, and perhaps allowing for N-Quads/TriG input as well as output). An updated playground architecture that allowed for changes like this more easily would be useful.

**Comment by dlongley** [comment]
We should add a link to https://playground.chapi.io/ in lieu of the signature tab. That will take people to another playground that is focused on signatures involving other standards work (signing / issuing JSON-LD-based VCs).

**Comment by dlongley** [comment]
A "Other Related Playgrounds" section could link to the CHAPI playground and a (future) YAML-LD playground.

File: playground/index.html [hunk 1]
```diff
@@ -518,12 +554,23 @@ <h4>Public Key for Verification</h4>
...
             </div>
             <div id="pane-signed-koblitz" class="tab-pane">
               <textarea id="signed-koblitz" class="codemirror-output"></textarea>
             </div>
+            -->
+            <div id="pane-signatures" class="tab-pane">
+              <div class="alert alert-info">
+                <strong>NOTE</strong>:
+                As of 2022-08-25 the former RSA and Bitcoin signature demos are
+                temporarilly unavailable. For a demo involving JSON-LD
```
Comment (TallTed) [inline playground/index.html:569]:
```suggestion
                As of 2022-08-25, the former RSA and Bitcoin signature demos are
                temporarily unavailable. For a demo involving JSON-LD
```

File: playground/index.html [hunk 2]
```diff
@@ -518,12 +554,23 @@ <h4>Public Key for Verification</h4>
...
               <textarea id="signed-rsa" class="codemirror-output"></textarea>
             </div>
             <div id="pane-signed-koblitz" class="tab-pane">
               <textarea id="signed-koblitz" class="codemirror-output"></textarea>
             </div>
+            -->
+            <div id="pane-signatures" class="tab-pane">
+              <div class="alert alert-info">
+                <strong>NOTE</strong>:
+                As of 2022-08-25, the former RSA and Bitcoin signature demos are
```
Comment (TallTed) [inline playground/index.html:568]:
```suggestion
                As of 2022-08-25, the demos of RSA and Bitcoin signatures are
```

**Pull request merged**